### PR TITLE
Feature/jet style

### PR DESCRIPTION
**Pull request by gannetson**

### CODE REVIEW

File: bluebottle/bluebottle_dashboard/static/jet/css/_changelist.scss [hunk 1]
```diff
@@ -394,20 +398,25 @@
...
+        cursor: pointer;
         border: 0;
         border-radius: 4px;
         height: 32px;
         line-height: 32px;
         outline: 0;
-        font-size: 12px;
-        font-weight: lighter;
+        font-size: 0.9rem;
+        font-weight: bold;
```
Comment (Pieter-Rees) [inline bluebottle/bluebottle_dashboard/static/jet/css/_changelist.scss]: Better declare font-weight in number. a.e. bold === 700

File: bluebottle/bluebottle_dashboard/static/jet/css/_dashboard.scss [hunk 2]
```diff
@@ -187,6 +185,9 @@ ul.actionlist li {
     border-radius: 4px;
     margin-bottom: 20px;
     transition: background $transitions-duration;
+    .float-right {
```
Comment (Pieter-Rees) [inline bluebottle/bluebottle_dashboard/static/jet/css/_dashboard.scss]: Better call it align-right. Float is counter intuitive

File: bluebottle/bluebottle_dashboard/static/jet/css/_forms.scss [hunk 3]
```diff
@@ -6,56 +6,55 @@
...
-    border: 0;
+    cursor: pointer;
     border-radius: 4px;
     height: 32px;
     line-height: 32px;
     outline: 0;
-    font-size: 12px;
-    font-weight: normal;
+    font-size: 0.9rem;
+    font-weight: bold;
```
Comment (Pieter-Rees) [inline bluebottle/bluebottle_dashboard/static/jet/css/_forms.scss]: Again better number then bold

File: bluebottle/bluebottle_dashboard/static/jet/css/_breadcrumbs.scss [hunk 4]
```diff
@@ -42,10 +41,13 @@ div.breadcrumbs {
 }
 
 .breadcrumbs {
+  a {
```
[the remark this answers is not in the document]
Comment (gannetson) [reply bluebottle/bluebottle_dashboard/static/jet/css/_breadcrumbs.scss]: I want to use as much templates from `django-jet` as is. Where possible I will use classes, but this `a` didn't have one :-(

File: bluebottle/bluebottle_dashboard/static/jet/css/_changelist.scss [hunk 5]
```diff
@@ -108,7 +108,8 @@
       margin-bottom: 5px;
       margin-right: 2px;
       vertical-align: top;
-
+      width: 321px;
+      background: #ffffff url('/static/assets/admin/img/search.svg') no-repeat 290px 50%;
```
[the remark this answers is not in the document]
Comment (gannetson) [reply bluebottle/bluebottle_dashboard/static/jet/css/_changelist.scss]: Check!

File: bluebottle/bluebottle_dashboard/static/jet/css/_dashboard.scss (hunk 2 above)
Comment (gannetson) [reply bluebottle/bluebottle_dashboard/static/jet/css/_dashboard.scss]: I was re-using an existing one, but you're right. I will create a new class.

File: bluebottle/bluebottle_dashboard/static/jet/css/_forms.scss (hunk 3 above)
Comment (gannetson) [reply bluebottle/bluebottle_dashboard/static/jet/css/_forms.scss]: Replaced all

File: bluebottle/bluebottle_dashboard/static/jet/css/_header.scss [hunk 6]
```diff
@@ -74,13 +75,16 @@
...
     top: 50%;
-    left: 20px;
-    margin-top: -8px;
     display: inline-block;
     font-size: 16px;
     cursor: pointer;
+    position: relative;
+    margin: 0;
+    left: 0;
+    border-radius: 12px;
```
[the remark this answers is not in the document]
Comment (gannetson) [reply bluebottle/bluebottle_dashboard/static/jet/css/_header.scss]: magic!

**Pull request merged**